packages/components/src/border-box-control/stories/index.tsx

```diff
@@ -35,72 +52,57 @@ const colors = [
 	{ name: 'Yellow 40', color: '#bd8600' },
 ];
 
-export default {
-	title: 'Components (Experimental)/BorderBoxControl',
-	component: BorderBoxControl,
-	parameters: {
-		knobs: { disable: false },
-	},
-};
+const Template: ComponentStory< typeof BorderBoxControl > = ( props ) => {
+	const { onChange, ...otherProps } = props;
+	const [ borders, setBorders ] = useState< AnyBorder >();
```

Comment:
No need to use the `AnyBorder` type, we can use the `value` prop type instead:

```suggestion
	const [ borders, setBorders ] = useState< typeof props[ 'value' ] >();
```

Reply:
✅ Done:  https://github.com/WordPress/gutenberg/pull/45002/commits/2edad2ea69b5c6ca137ff5eeff094ae9bb1365d9